base/sparse/cholmod.jl

```diff
@@ -87,22 +87,22 @@ function __init__()
             tmp = Array(Cint, 3)
             ccall((:cholmod_version, :libcholmod), Cint, (Ptr{Cint},), version_array)
```

Comment:
BTW, I don't understand why this line is here. `version_array` is declared `const` above, and already set during compilation using the very same call. Since it's supposed to hold the version used when building Julia, resetting it on init doesn't make any sense to me. Should I remove it?